### PR TITLE
[13.0][FIX] l10n_nl_location_nuts: mocked tests

### PR DESCRIPTION
**Pull request by astirpe**
Fixes error described in https://github.com/OCA/l10n-netherlands/pull/268#issuecomment-787965251

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# for tests
+vcrpy==2.0.1
```
Comment (StefanRijnhart) [inline requirements.txt]: What's the reason you picked an old version? There is now https://pypi.org/project/vcrpy/4.1.1/. If you have to specify a version at all, you could use `vcrpy<5.0.0` to protect against breaking API changes in the next major release.

Comment (astirpe) [reply requirements.txt]: I just copied it from another repo. I thought there was a reason for specifying that particular version.
Changed to `vcrpy<5.0.0`. Thanks!

**Review by StefanRijnhart — APPROVED** [review]

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**